modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/WebDataFile_conf.pm

```diff
@@ -0,0 +1,160 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=pod
+
+=head1 NAME
+
+Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf
+
+=head1 DESCRIPTION
+
+This pipeline automate current manual processing of the 7 species (https://2020.ensembl.org/app/species-selector) currently available on 2020 website.
+
+registry=${BASR_DIR}/ensembl-production_private/release_coordination/registries/reg_ens-sta-1_rw.pm
+HIVE_SRV=mysql-ens-hive-prod-1-ensadmin
+init_pipeline.pl Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf \
+ -registry ${registry} \
+ $($HIVE_SRV details hive) \
+ -pipeline_name Webdatafile_${ENS_VERSION} \
+ -release ${ENS_VERSION} \
+ -output_path /hps/nobackup2/production/ensembl/${USER}/test_webdatafile \
+ -hive_force_init 1 \
+ -run_all 1 \
+ -app_path ${BASE_DIR}/e2020_march_datafiles/ \
+ -genomeinfo_yml ${BASE_DIR}/e2020_march_datafiles/common_files/genome_id_info.yml \
+ -email ${USER}@ebi.ac.uk
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf;
+use strict;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+
+use Data::Dumper;
+
+sub default_options {
+    my $self = shift;
+    return {
+        %{$self->SUPER::default_options()},
+        species         => [],
+        division        => [],
+        antispecies     => [],
+        run_all         => 0, #always run every species
+        step             => [],  
+        genomeinfo_yml    => undef,
+        # Email Report subject
+        email_subject => $self->o('pipeline_name').'  pipeline has finished',
+        release => software_version()
+	};
+}
+
+sub pipeline_wide_parameters {
+    my $self = shift;
+    return { %{$self->SUPER::pipeline_wide_parameters()},
+        output_path       => $self->o('output_path'),
+        app_path          => $self->o('app_path'),
+        genomeinfo_yml    => $self->o('genomeinfo_yml')
+    };
+}
+
+sub pipeline_analyses {
+    my $self = shift;
+
+    return [
+        
+        { 
+          -logic_name  => 'GeneInfo_logic',
+          -input_ids  => [ {} ],
+          -module      =>  'Bio::EnsEMBL::Production::Pipeline::Webdatafile::GenomeInfoYml',
+          -parameters => { species => $self->o('species'),
+                           genomeinfo_yml => $self->o('genomeinfo_yml'), 
+                           run_all        => $self->o('run_all'),   
+                         },
+          -flow_into   =>  {  '2->A' => ['StepBootstrap'],
+                              '3->A'  => ['SpeciesFactory'],   
+                               # WHEN(
```

Comment:
Maybe this comment can be cleaned up now?

Reply:
resolved